features/step_definitions/helper_services.rb

```diff
@@ -304,7 +304,7 @@
   end
 
   #@result = user.cli_exec(:create, f: "https://raw.githubusercontent.com/openshift/origin/master/examples/gitserver/gitserver-ephemeral.yaml")
-  @result = user.cli_exec(:run, name: "git-client", image: "quay.io/openshifttest/origin-gitserver@sha256:8062457c330f0da521b340f2dceb13c4852d46ec58e45a9ef276a5ec639a328c", env: 'GIT_HOME=/var/lib/git')
+  @result = user.cli_exec(:run, name: "git-client", image: "quay.io/openshiftteststaging/origin-gitserver@sha256:4049e4bc7f38aee73701323c1f54df782a5cd2c337507e268cdba109d7cf172d", env: 'GIT_HOME=/var/lib/git')
```

Comment:
Do we sync this image in disconnect test?
I may miss something, could you tell me why we use openshiftteststaging repository but not openshfittest?

Reply:
The image will land in the openshifttest quay.io organization before merging together with the proper PR in flexy-templates.

Reply:
Thanks for your clarification